include/CL/opencl.hpp

```diff
@@ -4586,6 +4613,73 @@ class Image : public Memory
         }
         return param;
     }
+
+#if defined(cl_ext_image_requirements_info)
+    static cl_int getRequirementsInfoExt(const Context &context,
```

Comment:
It still bugs me a little that this is a static function of the `Image` class whereas `getSupportedImageFormats` is a member function of the `Context` class, but it's mostly the inconsistency that bugs me.

Maybe we should consider adding a static function for getSupportedImageFormats and a member function for getRequirementsInfoExt?  There's no reason we can't have both functions in both places.

Reply:
I will move it to the Context class.

Reply:
Done thanks !